plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceRuntimeInfo.java

```diff
@@ -355,4 +378,18 @@ public String projectsRoot() {
 
         return serversConf;
     }
+
+    /**
+     * Checks environment variable {@code CHE_DOCKER_USE_INTERNAL_CONTAINER_ADDRESS} to determine if
+     * direct addresses of containers should be used.
+     * @return true if {@code CHE_DOCKER_USE_INTERNAL_CONTAINER_ADDRESS} is "true", false otherwise.
+     */
+    protected boolean useInternalContainerAddresses() {
+        String useInternalContainerAddresses = System.getenv(CHE_DOCKER_USE_INTERNAL_CONTAINER_ADDRESS);
```

Comment:
1) I submitted a PR to fix that. I'm going to merge it soon. #2454 
2) Probably it was implemented before we merged feature with automatic conversion. Or we missed it in PR review. 
I believe that it should be fixed. @amisevsk If you want you can fix it also.
@TylerJewell should env variables be renamed to match properties names or it is better to try to use new property aliases feature?